projects/kubernetes/kube-master.yml

```diff
@@ -51,18 +51,19 @@ services:
      - /dev:/dev
      - /etc/resolv.conf:/etc/resolv.conf
      - /lib/modules:/lib/modules
+     - /run:/run:rshared,rbind
```

Comment:
there is no real need for `/run` to be `rshared,rbind` as no one ever mounts anything in `/run` AFAIK